content/post/retrospective2022/index.md

```diff
@@ -0,0 +1,258 @@
+  La _data science_ a beaucoup fait parler d'elle en 2022,
+  notamment du fait des deux coups
+  médiatiques d'[openAI](https://openai.com/), à savoir [`Dall-E`](https://openai.com/dall-e-2/)
+  et [`chatGPT`](https://openai.com/blog/chatgpt/),
+  qui ont créé un réel engouement au delà de la sphère
+  des praticiens de la donnée. Cet article revient 
+  ouvre une série de _post_ rétrospectifs sur l'année 2022
+  pour faire un bilan de l'actualité de la _data science_
+  et du réseau. 
+# Link this post with a project
+projects: []
+
+# Date published
+date: '2022-12-13T00:00:00Z'
+
+# Date updated
+lastmod: '2022-09-19T00:00:00Z'
+
+# Is this an unpublished draft?
+draft: false
+
+# Show this page in the Featured widget?
+featured: false
+
+# Featured image
+# Place an image named `featured.jpg/png` in this page's folder and customize its options here.
+image:
+  focal_point: ''
+  placement: 2
+  preview_only: false
+
+authors:
+  - linogaliana
+
+tags:
+  - NLP
+
+categories:
+  - Insee
+---
+
+La fin de l'année est généralement synonyme de
+bétisiers, _best of_ ou
+rétrospectives personnalisées qui nous permettent
+de nous rappeler les événements marquants de l'année.
+
+Pour célébrer la fin de l'année 2022, la _newsletter_
+de janvier adopte un format un peu spécial pour
+proposer, en deux temps, deux rétrospectives.
+
+Cette première _newsletter_ revient 
+sur les principaux événements de l'année 2022 dans le monde de
+la _data science_. La seconde _newsletter_ proposera
+une rétrospective quantitative sur le
+réseau des _data scientists_ de la statistique publique, à la manière
+des rétrospectives personnalisées de nos applications préférées.
+
+
+# Les IA créatrices de contenu à l'honneur
+
+Si l'année 2022 a été particulièrement riche dans le domaine
+de la _data science_, c'est principalement grâce à deux coups
+médiatiques d'[OpenAI](https://openai.com/),
+à savoir [`Dall-E`](https://openai.com/dall-e-2/)
+et [`ChatGPT`](https://openai.com/blog/chatgpt/). 
+
+Ces deux outils ont
+beaucoup fait parlé d'eux, au-delà
+de la sphère traditionnelle de la _data science_.
+Le _buzz_ a été intense sur `Twitter`, racheté en fin d'année par Elon Musk,
+ou sur `Mastodon`, 
+le réseau social qui a connu, en réaction,
+une croissance importante de son nombre d'utilisateurs.
+
+![](https://i.chzbgr.com/full/9717138688/hC5E738A3/chatgpt-will-see-all-criticisms-its-training-data-on-next-update-some-are-not-surviving-uprising)
+
+Cependant, ces innovations, parce qu'elles pourraient
+avoir des effets à long terme sur la manière dont le grand public
+appréhende l'intelligence artificielle, 
+ont beaucoup intéressé les médias traditionnels, notamment [_The Economist_ et sa _"Nouvelle Frontière"_](https://www.economist.com/news/2022/06/11/how-a-computer-designed-this-weeks-cover) ou [le _Guardian_ qui s'interroge](https://www.theguardian.com/commentisfree/2022/dec/10/i-wrote-this-column-myself-but-how-long-before-a-chatbot-could-do-it-for-me)).
```

Comment:
Comme l'article de The Economist n'est pas en accès libre, je pense que cela serait utile de dire une phrase sur le contenu.